### PR TITLE
SDK에서 web3-shh, web3-bzz 버전 업데이트

### PR DESCRIPTION
**Pull request by BoBeenLee**

### CODE REVIEW

**Comment by BoBeenLee** [comment]
https://www.notion.so/haechilabs/SDK-web3-shh-web3-bzz-c55ce0139cda44b29c236f14c75436fd

**Comment by BoBeenLee** [comment]
solution: https://github.com/ChainSafe/web3.js/issues/3898#issuecomment-774769469

File: packages/core/src/eth/coin.ts [hunk 1]
```diff
@@ -18,7 +18,9 @@ export abstract class Coin {
 
   protected constructor(coinData: CoinData) {
     this.coinData = coinData;
-    this.walletContract = new new Web3().eth.Contract(walletAbi as AbiItem[]);
+    this.walletContract = new new Web3().eth.Contract(
```
Comment (cloudinertia) [inline packages/core/src/eth/coin.ts:21]: could you explain why you typecast walletContract to "any"?

Comment (BoBeenLee) [reply packages/core/src/eth/coin.ts]: fixed it @cloudinertia

**Review by cloudinertia — APPROVED** [review]

**Review by hihiboss — APPROVED** [review]

**Pull request merged**